src/main.rs

```diff
@@ -70,19 +52,28 @@ async fn main() -> Result<()> {
         None => return Err(anyhow!("could not build plugin")),
     };
 
+    let config_path = Path::new(&plugin.configuration().lightning_dir).join("backup.toml");
```

Comment:
I ***hate*** that we abandon the CLN config and switch to our own config file which *has to be at a hardcoded location*